protocol/x/clob/keeper/deleveraging.go

```diff
@@ -315,15 +260,12 @@ func (k Keeper) OffsetSubaccountPerpetualPosition(
 			1,
 			types.ModuleName, metrics.CheckTx, metrics.Deleveraging, metrics.NotEnoughPositionToFullyOffset, metrics.Count,
```

Comment:
consider modifying these metrics to tell a clearer story. There is a difference between not finding anything to deleverage against vs finding partial positions to deleverage against vs finding a full fill (deltaQuantumsRemaining = 0)